### ISSUE TITLE
evaluation and accuracy

### ISSUE THREAD

**Issue by ylqi007** [balancap/SSD-Tensorflow#334]
I have two questions about the evaluation, could anyone give me some hint and help, please?

1. AP_VOC07 v.s. AP_VOC12
When I ran the `eval_ssh_network.py`, I got two results, `AP_VOC07/mAP` and `AP_VOC12/mAP`. But in the original paper and this repo, only one result is shown. Which kind of result is shown in the original paper and this repo, `AP_VOC07/mAP` or `AP_VOC12/mAP`?

2. I didn't get the result like the author shown in this repo.
When I finetuned on `VOC07 train` dataset based on `ssd_300_vgg.ckpt`, and evaluated on `VOC07 test` dataset, I got the result like bellow, and the `AP_VOC07/mAP` matches the result shown in the SSD paper,
![PASCAL_VOC_2007_20190301_Original_Losses_12000](https://user-images.githubusercontent.com/23485976/54630714-74409100-4a48-11e9-96fa-e0036c4632c0.png)
But when I finetuned on `VOC07+12 trainval` dataset based on `ssd_300_vgg.ckpt`, and evaluated on `VOC07 test` dataset, I got the result like bellow, and the results, both `AP_VOC07` and `AP_VOC12`, are different from the result (*0.778*) shown in this repo, 
![PASCAL_VOC_0712_20190317_Original_17729](https://user-images.githubusercontent.com/23485976/54630802-a8b44d00-4a48-11e9-8a3c-f985d2c4c7c1.png)

This is the result from this repo
![image](https://user-images.githubusercontent.com/23485976/54630981-08aaf380-4a49-11e9-843c-6008602172a5.png)

I am confusing now. Could anyone give me some hint and help?

Sincerely

**Comment by fuaiguo** [comment]
Hello, I would like to know whether the code you used is balancap/ ssd-tensorflow. I also did the same repetition, but the effect was very poor and the mAP was very low.I would like to ask you, what changes have you made to the code?thank you @ylqi007

**Comment by ylqi007** [comment]
@fuaiguo 
The code is from balancap/ssd-tensorflow. The training script is shown as below:

```
CHECKPOINT_PATH=./checkpoints/ssd_300_vgg.ckpt
DATASET_DIR=/tmp/tfrecords_pascal_2007
TRAIN_DIR=./logs/PASCAL_VOC_2007/

python3 train_ssd_network.py \
    --train_dir=${TRAIN_DIR} \
    --dataset_dir=${DATASET_DIR} \
    --dataset_name=pascalvoc_2007 \
    --num_classes=21 \
    --dataset_split_name=train \
    --model_name=ssd_300_vgg \
    --save_summaries_secs=60 \
    --save_interval_secs=600 \
    --checkpoint_path=${CHECKPOINT_PATH} \
    --optimizer=adam \
    --max_number_of_steps=7000 \
    --learning_rate=0.001 \
    --batch_size=32 \
    --match_threshold=0.5
```

**Comment by fuaiguo** [comment]
By the way, have you modified the original code? I think there are some differences between the code and the paper. According to the original code, the effect is very poor.

**Comment by ylqi007** [comment]
@fuaiguo 
No. I used the original code.

**Comment by lloydnguyen96** [comment]
I also got a really low mAP
AP_VOC07/mAP[1.7213194274951696e-05]
AP_VOC12/mAP[1.1840413668554323e-05]
Could anyone please help me with this!!!

**Comment by angleflower** [comment]
> I also got a really low mAP

> Could anyone please help me with this!!!

**Comment by navy63** [comment]
CHECKPOINT_PATH=./checkpoints/ssd_300_vgg.ckpt
checkpoint_path=./checkpoints/vgg_16.ckpt
This is the essence.
